src/latexify/ast_utils.py

```diff
@@ -147,3 +147,9 @@ def extract_function_name_or_none(node: ast.Call) -> str | None:
         return node.func.attr
 
     return None
+
+
+def ast_function_def(*args, **kwargs) -> ast.FunctionDef:
```

Comment:
Please follow the convention of this library:

* function names should be basically starting with a verb
* add docstring (with Google's docstring style)
* add unit test for this function (in `ast_utils_test.py`).
* (if possible) full argument list is encouraged rather than `args` and `kwargs`.

Reply:
Yes, perfect. I updated the coding convention and created the test case.
Kwargs current can't replace be cause `type_params` is not optional so we need pass through kwargs and filter in the wrapper function